From: Gunnar Wolf <gwolf@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2009-12-03
Description: Sets the daemon's default user/group to www-data

Index: cherokee/cherokee.conf.sample.pre
===================================================================
--- cherokee.orig/cherokee.conf.sample.pre	2009-12-03 11:22:28.000000000 -0600
+++ cherokee/cherokee.conf.sample.pre	2009-12-03 11:22:32.000000000 -0600
@@ -18,6 +18,8 @@
 server!server_tokens = full
 server!panic_action = %prefix%/share/cherokee/cherokee-panic
 server!pid_file = %localstatedir%/run/cherokee.pid
+server!user = www-data
+server!group = www-data
 
 # Default virtual server
 #
